src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -804,6 +807,13 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 		return this.model.getEditors(order, options);
 	}
 
+	findEditors(resource: URI, order: EditorsOrder, options?: { excludeSticky?: boolean }): EditorInput[] {
+		const canonicalResource = this.uriIdentityService.asCanonicalUri(resource);
+		return this.getEditors(order, options).filter(editor => {
```

Comment:
Looking at the users, we seem to not require the `order` for this method?

Reply:
What do you mean we don't require order?

Reply:
I was wondering if `EditorsOrder` needs to be passed into `findEditors` at all if everyone asks for the same order

Reply:
Ah I guess not. I'll fix that, I just figured for verbosity you could choose if you really wanted to.